utils/zerovec/src/varzerovec/databake.rs

```diff
@@ -14,11 +14,46 @@ impl<T: VarULE + ?Sized> Bake for VarZeroVec<'_, T> {
     }
 }
 
-impl<T: VarULE + ?Sized> Bake for &VarZeroSlice<T> {
+impl<T: VarULE + ?Sized> Bake for VarZeroSlice<T> {
```

Comment:
Question: Not sure I understand this part. Note that this impl returns a `&VarZeroSlice<T>` because it is a DST. Shouldn't the impl and test be on `&VarZeroSlice<T>` only? Note that `VarZeroSlice::from_bytes_unchecked` returns a `&`.

For example, you have

```rust
    test_bake!(
        VarZeroSlice<str>,
        const: unsafe {
            crate::VarZeroSlice::from_bytes_unchecked(&[
                2u8, 1u8, 0u8, 22u8, 0u8, 77u8, 1u8, 92u8, 17u8,
            ])
        },
        zerovec
    );
```

I think this expands to something that includes

```rust
const _: &VarZeroSlice<str> = &crate::VarZeroSlice::from_bytes_unchecked(...);
```

But this doesn't make sense because the left side is `&VarZeroSlice<str>` and the right side is `&&VarZeroSlice<str>`

Clearly it works, because CI is happy, so I'm missing something.

Reply:
No idea why this worked either